src/pages/LandingPage.py

```diff
@@ -23,12 +20,3 @@ def __init__(self, driver):
         self.create_event_btn = ButtonElement(self.CREATE_EVENT_BTN_CSS, driver)
         self.join_eventsexpress_btn = ButtonElement(self.JOIN_EVENTSEXPRESS_BTN_CSS, driver)
         self.log_out_btn = ButtonElement(self.LOG_OUT_BTN_CSS, driver)
-
-    def scroll_down_page(self):
-        return self.scroll_down(self.All_PAGE_CSS)
-
-    def get_upcoming_public_event(self):
-        self.scroll_down_page()
-        return self.find_element_by_css(self.UPCOMING_PUBLIC_EVENT)
-
```

Comment:
Leave only one empty line between methods and at the bottom of files.

Reply:
Resolved in the new commit